File: number-format/seth.py
#!/usr/bin/env python

from sys import argv

x, y = str(float(argv[1])).split('.')
x2 = ''

for i, c in enumerate(x[::-1]):
    if i % 3 == 0:
        x2 += ','
    
    x2 += c

print '.'.join([x2[::-1].strip(','), y])

File: number-format/seth.rb
#!/usr/bin/env ruby

x, y = ARGV[0].to_f.to_s.split('.')
x2 = ''

x.reverse.chars.each_with_index do |c, i|
  if i % 3 == 0
    x2 += ','
  end
  
  x2 += c
end

puts [x2.reverse.chomp(','), y].join('.')
